bitrise.yml

```diff
@@ -127,6 +129,23 @@ workflows:
         title: Run SonarQube Scanner
         inputs:
         - content: ./gradlew sonarqube
+  _create-github-release:
+    steps:
+    - script@1:
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -ex
+
+            _CHANGELOG="${GIT_REPOSITORY_URL%.git}/tree/$BITRISE_GIT_TAG#changelog"
```

Comment:
Will this only include the changelog for the released version?

Reply:
Yes, sample is in the [releases](https://github.com/rakutentech/android-sdkutils/releases) page